integration-tests/src/test/resources/features/action-sendtobridge.feature

```diff
@@ -0,0 +1,105 @@
+Feature: SendToBridge Action tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "bridge1"
+    And the Bridge "bridge1" is existing with status "ready" within 2 minutes
```

Comment:
Maybe it would be good to use some random suffix for the name of the bridge. It may happen that two users are running the same tests simultaneously on the external cluster or user and some Github action is running same test at one moment.

Reply:
Under the hood the Bridge name is generated and validated for uniqueness. In the scenario definition the value is used as a placeholder. So the Bridge name written in scenario as `bridge1` is translated to `test-<4 numbers from UUID>`.